ferretdb_packaging/defineversion/main.go

```diff
@@ -0,0 +1,235 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package main defines debian version number of DocumentDB for CI builds.
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"regexp"
+	"slices"
+	"strings"
+
+	"github.com/sethvargo/go-githubactions"
+)
+
+func main() {
+	controlFileF := flag.String("control-file", "../pg_documentdb/documentdb.control", "pg_documentdb/documentdb.control file path")
+
+	flag.Parse()
+
+	action := githubactions.New()
+
+	if *controlFileF == "" {
+		action.Fatalf("%s", "-control-file flag is empty.")
+	}
+
+	version, err := controlVersion(*controlFileF)
+	if err != nil {
+		action.Fatalf("%s", err)
+	}
+
+	debugEnv(action)
+
+	res, err := define(version, action.Getenv)
+	if err != nil {
+		action.Fatalf("%s", err)
+	}
+
+	setResults(action, res)
+}
+
+// controlDefaultVer matches the default_version field in control file,
+// see pg_documentdb_core/documentdb_core.control.
+var controlDefaultVer = regexp.MustCompile(`default_version\s*=\s*'([0-9]+\.[0-9]+-[0-9]+)'`)
```

Comment:
Let's add `$` and `^` to be sure.
We probably could rely on exactly one space around `=`.
And let's use named capture groups: major, minor, patch.